### PR TITLE
update aimec for using DEM with different spatial resolution

### PR DESCRIPTION
**Pull request by awirb**

### CODE REVIEW

**Review by fso42 — APPROVED** [review]

File: docs/moduleAna3AIMEC.rst [hunk 1]
```diff
@@ -25,6 +25,11 @@ Inputs
 *  Results from avalanche simulation (when using results from com1DFA,
    the helper function :py:func:`ana3AIMEC.dfa2Aimec.mainDfa2Aimec` in :py:mod:`ana3AIMEC.dfa2Aimec` fetches and prepares the input for Aimec)
 
+.. Note:: If the results have been performed by remeshing the DEM to a different spatial resolution, it is still
+          possible to provide a DEM of a different spatial resolution compared to the result files to aimec.
+          However, the z coordinates used to compute the location of the runout point and the fall height (deltaH)
+          are then derived from the provided dem data.
```
Comment (matthiasto) [inline docs/moduleAna3AIMEC.rst:31]: I would write:
The DEM resolution and extend can be different from the result raster ones (but all result raster extend and resolution need to be identical). The DEM file is required in to compute the z coordinate of the avalanche path as well as to compute de fall height (deltaH) of the avalanche.

File: avaframe/runScripts/runComparisonModules.py [hunk 2]
```diff
@@ -134,6 +136,7 @@
             else:
                 cfgAimec['FLAGS']['flagMass'] = 'False'
             cfgAimec['AIMECSETUP']['comModules'] = 'com1DFAOrig|com1DFA'
+            cfgAimec['AIMECSETUP']['resType'] = 'ppr'
```
Comment (matthiasto) [inline avaframe/runScripts/runComparisonModules.py:140]: why do you hardcode this? why not reed the ini file?

**Review by fso42 — APPROVED** [review]

**Review by matthiasto — CHANGES_REQUESTED** [review]
I havent tried to run it, will try when I am finished with my PR

File: avaframe/runScripts/runComparisonModules.py (hunk 2 above)
Comment (awirb) [reply avaframe/runScripts/runComparisonModules.py:140]: this if for the report generation of the comparison report - where we use the locals for com1DFA and com1DFAOrig for this runScript to perform the ava simulations, but to run aimec and generate the report we use some fixed settings in the  runScripts - that is for this runScript one doesn't have to change the aimec configuration - but it is set how we defined it for this test

File: docs/moduleAna3AIMEC.rst (hunk 1 above)
Comment (awirb) [reply docs/moduleAna3AIMEC.rst]: I updated this sentence

File: docs/moduleAna3AIMEC.rst [hunk 3]
```diff
@@ -25,6 +25,12 @@ Inputs
 *  Results from avalanche simulation (when using results from com1DFA,
    the helper function :py:func:`ana3AIMEC.dfa2Aimec.mainDfa2Aimec` in :py:mod:`ana3AIMEC.dfa2Aimec` fetches and prepares the input for Aimec)
 
+.. Note:: The spatial resolution of the DEM and its extend can differ from the result raster data.
+          However, all the result raster have to be of identical spatial resolution and extent.
+          This might be of interest if the result rasters have been derived by remeshing the DEM to different spatial
+          resolution. The DEM data is used to compute the z coordinate of the location of the runout point as well as
+          the fall height (deltaH).
+
```
Comment (matthiasto) [inline docs/moduleAna3AIMEC.rst:33]: I like this

File: avaframe/runScripts/runComparisonModules.py (hunk 2 above)
Comment (matthiasto) [reply avaframe/runScripts/runComparisonModules.py:140]: mm still not convinced

**Review by fso42 — APPROVED** [review]

**Pull request merged**